templates/components/multiagent/typescript/workflow/stream.ts

```diff
@@ -46,15 +45,16 @@ export async function createStreamFromWorkflowContext<Input, Output, Context>(
         closeStreams(controller);
         return;
       }
-      const text = trimStartOfStream(chunk.delta ?? "");
-      if (text) {
-        controller.enqueue(encoder.encode(text));
+      if (chunk.delta) {
+        controller.enqueue(encoder.encode(chunk.delta));
       }
     },
   });
 
   return {
-    stream: mainStream.pipeThrough(createStreamDataTransformer()),
+    stream: mainStream
+      .pipeThrough(createStreamDataTransformer())
+      .pipeThrough(new TextDecoderStream()),
```

Comment:
if you use `LlamaIndexAdapter`, it takes care of piping dataStream through the text stream
```suggestion
    stream: mainStream
```